fe/fe-core/src/main/java/com/starrocks/mysql/privilege/AuthUpgrader.java

```diff
@@ -270,9 +272,12 @@ protected void upgradeUserGlobalPrivileges(
                     upgradeDbPrivileges(STAR, privilege, collection, null);
                     upgradeTablePrivileges(STAR, STAR, privilege, collection, null);
                     upgradeViewPrivileges(STAR, STAR, privilege, collection, null);
+                    upgradeMaterializedViewPrivileges(STAR, STAR, privilege, collection, null);
                     break;
 
                 case ADMIN_PRIV:
+                    upgradeDbPrivileges(STAR, privilege, collection, null);
```

Comment:
we don't need to upgrade ADMIN_PRIV to specific privilege, ADMIN_PRIV is upgraded to built-in role, plz read the related source code before you modify, 
![image](https://user-images.githubusercontent.com/107105845/206988825-8c3a7f20-a4e5-430b-b786-185f1aa8d3a0.png)

Reply:
and if you still have any question here, feel free to call me